src/main/java/com/github/knaufk/flink/faker/FakerUtils.java

```diff
@@ -17,11 +17,11 @@
   private static DateTimeFormatter formatter =
       DateTimeFormatter.ofPattern(FAKER_DATETIME_FORMAT, new Locale("us"));
 
-  static Object stringValueToType(String value, LogicalTypeRoot logicalType) {
+  static Object stringValueToType(String[] stringArray, LogicalType logicalType) {
+    String value = stringArray.length > 0 ? stringArray[0] : "";
```

Comment:
It is needed in some cases like when the SQL query defines a row with an empty schema `ROW<>`. I am not sure whether we want to support this, but I thought it could be a supportable edge case. What do you think?